specs/pvs/corla.pvs

```diff
@@ -438,8 +461,10 @@ BEGIN
 
 % @description A decision to be made, either the selection of one or
 % more candidates for an office or the choice of a course of action.
+% @todo kiniry Need to clarify choices vs options vs votes.
   ballot_contest: TYPE = contest
-  choice: TYPE
+  option: TYPE = string
+  choice: TYPE = set[option]
```

Comment:
Right now option is abstracted to a string, which can, of course, encode anything.  Right now I have encoded a voter choice as a set of options, thus this model only handles standard multi-seat plurality voting schemes.